crates/store/src/genesis/config/mod.rs

```diff
@@ -66,14 +68,14 @@ impl Default for GenesisConfig {
             .expect("Timestamp should fit into u32"),
             wallet: vec![],
             fee_parameters: FeeParameterConfig {
-                symbol: "MIDEN".to_owned(),
+                symbol: TokenSymbolStr::from_str("MIDEN").unwrap(),
```

Comment:
We could make this an associated `const` though maybe this is less useful once the native faucet becomes its own type (since the constant would likely be there instead? maybe?).

Reply:
Opened https://github.com/0xMiden/miden-base/issues/1738.

Reply:
I'd leave this for a future PR